src/redux/books/books.js

```diff
@@ -1,30 +1,54 @@
-const initialState = [{ title: 'tarik', author: 'bouari', id: 1 },
-  { title: 'dona', author: 'persie', id: 2 }];
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
 
-const ADD_BOOK = 'redux/books/book/ADD_BOOK';
-const RMV_BOOK = 'redux/books/book/RMV_BOOK';
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const RMV_BOOK = 'bookstore/books/RMV_BOOK';
+const GET_BOOK = 'bookstore/books/GET_BOOKS';
+const key = 'HNRWWKcb5eZvN9Tn2PtR';
+const urlApi = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${key}/books`;
 
-export const bookReducer = (state = initialState, action) => {
+export const bookReducer = (state = [], action) => {
   switch (action.type) {
-    case ADD_BOOK:
-      return [
-        ...state,
-        action.book,
-      ];
-    case RMV_BOOK:
-      return state.filter((item) => item.id !== action.id);
+    case 'bookstore/books/GET_BOOK/fulfilled':
+      return action.payload.book;
+    case 'bookstore/books/ADD_BOOK/fulfilled':
+      return [...state, action.payload.book];
+    case 'bookstore/books/RMV_BOOK/fulfilled':
+      return [...state.filter((item) => item[0] !== action.payload.id)];
 
     default:
       return state;
   }
 };
 
-export const enterBook = (book) => ({
-  type: ADD_BOOK,
-  book,
+export const getBooks = createAsyncThunk(GET_BOOK, async () => {
+  const res = await axios.get(urlApi);
+  return { book: Object.entries(res.data) };
 });
 
```

Comment:
- [ ] Good job implementing the `getBooks` function. It does work fine and upon page load, it fetches the books from the API alright, I confirmed that in the network requests, I saw the list of all the books added to the API, but they are not displayed on the page. Kindly fix that to get your project approved.